spel/scripts/amigen7-build.sh

```diff
@@ -220,6 +219,54 @@ function CollectManifest {
     rpm --root "${CHROOT}" -qa | sort -u >> /tmp/manifest.txt
 }
 
+function ComposeAWSutilsString {
+    # Construct the cli option string for aws utils
+    CLIOPT_AWSUTILS=("-m ${CHROOT}" "-d ${ELBUILD}/AWSpkgs")
+
+    # Whether to install AWS CLIv1
+    if [[ -n "${AWSCLIV1SOURCE}" ]]
+    then
+      CLIOPT_AWSUTILS+=("-C ${AWSCLIV1SOURCE}")
+    fi
+
+    # Whether to install AWS CLIv2
+    if [[ -n "${AWSCLIV2SOURCE}" ]]
+    then
+      CLIOPT_AWSUTILS+=("-c ${AWSCLIV2SOURCE}")
+    fi
+
+    # Epel mirrors are maddening; retry 5 times to work around issues
+    echo "Executing AWSutils.sh"
+    retry 5 bash -eux -o pipefail "${ELBUILD}/AWSutils.sh ${CLIOPT_AWSUTILS[*]}"
```

Comment:
Similar to the other function, `ComposeCliOptionString`, will it work if we leave the three lines above in the "main" part of the workflow? This function `ComposeAWSutilsString` would literally just set the variables, and calling `AWSutils.sh` would happen outside the function...

Reply:
Moved the lines to call the AWSutils.sh script back into main and out of the function that is meant to set the variables.